demo

```diff
@@ -22,6 +22,10 @@ if [ "$#" -eq 0 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
     exit 1
 fi
 
+# For the integration tests the name of the docker-compose project is
+# generated by pytest
+DOCKER_COMPOSE_PROJECT_NAME="${DOCKER_COMPOSE_PROJECT_NAME:-integration}"
```

Comment:
I would put this and the related changes in a different commit. It's not related to Mac at all, is it?

Reply:
No, it was related to the tests. KK, I'm sorry, I will seperate it.